### DISCUSSION TITLE
Can not control organization but can visit some parts of settings

### DISCUSSION THREAD

**Discussion by hwpplayer1** [community/community#21600] [New to GitHub]
<p>I can not control organization but can visit some parts of settings, like i can delete verified organization domain, and i can get access to projects,</p>
<p>I tried to delete the organization but now I see still it is working but broken ?</p>
<p>Do you know why ? I asked for support with a ticket Thanks !..</p>
<p>Which is why I offer account delete protection what do you think about that ?</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I can confirm we have received your tickets and I believe private support is the best place to address your issue.</p>
<p>Please do not open more tickets - this will just delay the time until response.</p>

**Comment by hwpplayer1** [comment]
<p>Sorry I could not be able to get access to this mission critical organizations yet. Can I open another ticket ? Thanks</p>

**Comment by canuckjacq** [comment, marked as answer]
<p><a class="mention" href="/hwpplayer1">@hwpplayer1</a> It looks as though you’ve had multiple replies. If you open another ticket, please be very specific about the problem you are having. Try saying “I want to do ______” and “When I try to do _____, this happens: _______”. The more specific you can be, the more likely it is that you’ll receive a helpful reply.</p>
<p>If you are not receiving the replies, please check and email filters or folders you have set up for that inbox.</p>

**Comment by hwpplayer1** [comment]
<p>Thanks, I am getting better and learning how to found an organization Thanks for your understanding</p>
